CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## v1.1.0 (YYYY-MM-DD)
+
+- Modify `docker-compose-developer` configuration files, and `Makefile`, to support running everything in containers for local development
```

Comment:
Not sure if we need to include this to the change log, given these are not user facing changes